LinearRegression/Blog/main.tex

```diff
@@ -0,0 +1,235 @@
+\subsubsection*{F-statistic}
+\begin{figure}[H]\includegraphics[width=0.5\linewidth]{27}\end{figure}
+Below $R^2$ and the adjusted $R^2$ is the F-statistic which is the statistic of an F-test checking whether our model fits the data well. The null hypothesis of the F-test is that the model with no predictors but only an intercept can fit the data as well as the proposed one. To put it simply, it's testing whether $R^2$ is significantly greater than zero. The F-statistic is the ratio of the explained variance to the unexplained variance, so a higher value of F-statistic is evidence against the null hypothesis.
+
+The fourth row on the right gives the p-value corresponding to the F-statistic. Our model yields a very small p-value of 5.12e-67, so there is sufficient evidence to reject the null hypothesis and conclude that our model does have some predictive power.
+\subsubsection*{Confidence Intervals and t-tests}
+\begin{figure}[H]\includegraphics[width=\linewidth]{28}\end{figure}
+Perhaps the most important information is located in the central part of the summary. We can see the estimated values of each coefficient in the first column, followed by an estimate of the standard deviation of the coefficient. The last two columns provide 95\% confidence intervals constructed from the coefficients and their standard errors in the first two columns.
+
+The two columns in the middle are the results of t-tests checking for the relevancy of each coefficient. The null hypothesis of each t-test is that the corresponding coefficient equals zero, meaning no linear relationship between this predictor and the target variable. The t-statistic is the value in the first column (mean) divided by the second column (standard error); the p-values corresponding to each t-statistic are presented in the next column. You can see that the p-values of all our coefficients except X1 are smaller than .01, so we can safely reject the null hypotheses and say that they are indeed relevant to predicting our target variable. However, X1 may be a redundant variable at .01 significance level.
+
+\section*{Stepwise Regression}
```

Comment:
I think that this section could definitely be shortened. For example, a sentence like "An easier predictor selection procedure, as I mentioned previously, is backwards stepwise regression" could be phrased as "Backwards stepwise regression is an easier procedure and reliable predictor".